modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/RasterMBLayer.java

```diff
@@ -225,10 +225,17 @@ public Number getFadeDuration() {
 
         // Use of builder is easier for code examples; but fills in SLD defaults
         // Currently only applies the opacity.
+        if(getBrightnessMax() != null || getBrightnessMin() != null){
```

Comment:
You seem to be using the raster-brightness-min and raster-brightness-max to define a contrast function, without handling the mbstyle raster-contrast property. I think you need to take into account both here.

In fact, I am not entirely certain the MapBox brightness contrast settings map especially well to ContrastMethod.Normalize in GeoTools (although I can't seem to find a better match for the functionality).

Given the poor documentation of the ContrastMethod stuff, you'll probably want to do some manual testing to see that this is behaving how you expect - have you done this yet?